spec/integ/rendezvous/MSC4108SignInWithQR.spec.ts

```diff
@@ -268,7 +268,8 @@ describe("MSC4108SignInWithQR", () => {
         it("should abort if device doesn't come up by timeout", async () => {
             jest.spyOn(global, "setTimeout").mockImplementation((fn) => {
                 (<Function>fn)();
-                return -1;
+                // TODO: mock timers properly
+                return -1 as any;
```

Comment:
Why are these needed now but weren't before?

Reply:
I'm not actually sure...it must be due to a dependency update.  Typechecks expect the return value to be a [NodeJS Timeout](https://nodejs.org/api/timers.html#class-timeout) instead of a timeout ID.